build.proj

```diff
@@ -162,6 +162,13 @@
     <CallTarget Targets="CodeSignBinaries" Condition=" '$(CodeSign)' == 'true' " />
     
     <Message Importance="high" Text="Building Setup..." />
+    <CallTarget Targets="BuildInstaller" 
+                Condition=" '$(CodeSign)' == 'true' and '$(Scope)' == 'all'" />
+  </Target>
+  
+    <!-- Build all flavors of the installers -->
+  <Target Name="BuildInstaller" DependsOnTargets="RestoreNugetPackages;BuildMsBuildTask">
+    <Message Importance="high" Text="Building Installers..." />    
     <MSBuild
```

Comment:
This task should be called as part of Build and its splitted now just to be called from BuildInstaller.ps1